src/sail_lean_backend/sail_plugin_lean.ml

```diff
@@ -154,26 +154,26 @@ let create_lake_project (out_name : string) =
   let project_dir = base_dir ^ "/" ^ out_name in
   Unix.mkdir project_dir 0o775;
   (* Create .gitignore file *)
-  let gitignore = open_out (project_dir ^ "/.gitignore") in
+  let gitignore = open_out (Filename.concat project_dir "/.gitignore") in
```

Comment:
`Filename.concat` will add the `/` (or `\` on windows) so the string should just be `.gitignore`. And the same thing for the `/lean-toolchain`  and lakefile lines below